Sources/NIOSSH/SSHPacketSerializer.swift

```diff
@@ -73,7 +73,7 @@ extension ByteBuffer {
         ///   byte[m]   mac (Message Authentication Code - MAC); m = mac_length
 
         /// payload
-        self.moveWriterIndex(forwardBy: 5)
+        self.writeMultipleIntegers(UInt32(0), UInt8(0))
```

Comment:
nit: there's also `writeRepeatedBytes` (or something similar to that) which might be a teeny tiny bit more obvious.

Reply:
Oh I actually thought this one made more sense given the packet format laid out above.

Reply:
Makes sense, I was looking at it as the diff from “move the writer by 5”.